lib/create-organization-api.js

```diff
@@ -15,6 +15,8 @@ import entities from './entities'
  * @prop {function} getAppDefinitions
  * @prop {function} getOrganizationMembership
  * @prop {function} getOrganizationMemberships
+ * @prop {function} getOrganizationInvitation
```

Comment:
@guilebarbosa made a good point about these being the only types of invitations we have in cf, but I still like this better. Sometimes the js sdk is the first way people interact with contentful, so it's good to be clear about what the invitation is for :)

Reply:
so nice job! :)